File: example/src/main/kotlin/com/example/_02_structural_patterns/_07_bridge/java/_01_before/Champion.java
package com.example._02_structural_patterns._07_bridge.java._01_before;

import com.example._02_structural_patterns._07_bridge.java._02_after.Skin;

public interface Champion extends Skin {

    void move();

    void skillQ();

    void skillW();

    void skillE();

    void skillR();
}

File: example/src/main/kotlin/com/example/_02_structural_patterns/_07_bridge/kt/_01_before/Champion.kt
package com.example._02_structural_patterns._07_bridge.kt._01_before

import com.example._02_structural_patterns._07_bridge.kt._02_after.Skin

interface Champion : Skin {
    fun move()
    fun skillQ()
    fun skillW()
    fun skillE()
    fun skillR()
}
